frontend/src/metabase/dashboard/components/DashboardHeader/buttons/SaveEditButton.tsx

```diff
@@ -0,0 +1,77 @@
+import cx from "classnames";
+import { msgid, ngettext, t } from "ttag";
+
+import ActionButton from "metabase/components/ActionButton";
+import ButtonsS from "metabase/css/components/buttons.module.css";
+import {
+  setEditingDashboard,
+  updateDashboardAndCards,
+} from "metabase/dashboard/actions";
+import { getMissingRequiredParameters } from "metabase/dashboard/selectors";
+import { useDispatch, useSelector } from "metabase/lib/redux";
+import { dismissAllUndo } from "metabase/redux/undo";
+import { Tooltip } from "metabase/ui";
+import type { UiParameter } from "metabase-lib/v1/parameters/types";
+
+export const SaveEditButton = (props: { actionFn: () => void }) => {
```

Comment:
should we rename `actionFn` here to `onDoneEditing`?

Reply:
yes!